test/webview/View.test.ts

```diff
@@ -318,11 +318,26 @@ describe('View', () => {
                         file: 'developer-tutorials/add-environment.md'
                     },
                     {
-                        title: 'Debug a smart contract',
-                        length: '30-40 mins',
-                        difficulty: 'moderate',
-                        description: 'Learn how to make iterative changes to your smart contract using the VS Code debugger',
-                        file: 'developer-tutorials/debug.md'
+                      title: 'Creating a new identity',
+                      length: '2-3 mins',
+                      difficulty: 'simple',
+                      description: 'Learn how to create your very own identity using the Local Fabric CA',
```

Comment:
maybe change the description to `Learn how to create new identities for the Local Fabric`

Reply:
Done